Jenkinsfile_parameterized

```diff
@@ -0,0 +1,17 @@
+#!groovy
```

Comment:
do you need this `Jenkinsfile_parameterized` generally teams don't use sandbox that much

Reply:
Nah I don't, will remove it now